sql/create_indicators_table.sql

```diff
@@ -1,48 +1,117 @@
 CREATE TABLE wow_indicators AS 
 
-	WITH hpd_viol AS (
+	WITH email_dates AS (
+		SELECT 
+			email_date_send::date,
+			(email_date_send - interval '1 day')::date AS email_date_end,
```

Comment:
oo another opportunity for refactoring the alerts_district sql and auth provider code if the dates are being built into the indicators table ! this is really great

Reply:
yeah I was thinking that too, but didn't want to touch anything in this PR

Reply:
yeah heard, i think it should be an entirely new PR too